### PR TITLE
Fixes and clean up for TPC-H examples

### PR DESCRIPTION
**Pull request by ehsantn**
Clean up like removing unnecessary code and fixed formatting.

### CODE REVIEW

File: examples/tpch/generate_data_pq.py [hunk 1]
```diff
@@ -35,15 +33,24 @@
...
     tables = {}
     tables["customer"] = ("c", num_pieces_base, load_customer)
-    tables["lineitem"] = ("L", num_pieces_base*10, load_lineitem_with_date)
-    tables["nation"] = ("n", 1, load_nation)  # * dbgen only produces one file for nation with SF1000 *
+    tables["lineitem"] = ("L", num_pieces_base * 10, load_lineitem_with_date)
+    tables["nation"] = (
+        "n",
+        1,
+        load_nation,
+    )  # * dbgen only produces one file for nation with SF1000 *
```
Comment (alifarhid) [inline examples/tpch/generate_data_pq.py:41]: @ehsantn any reason each entry in this tuple is broken into new lines?

File: examples/tpch/generate_data_pq.py [hunk 2]
```diff
@@ -35,15 +33,24 @@
...
     tables["part"] = ("P", num_pieces_base, load_part)
     tables["partsupp"] = ("S", num_pieces_base, load_partsupp)
-    tables["region"] = ("r", 1, load_region)  # * dbgen only produces one file for region with SF1000 *
-    tables["supplier"] = ("s", num_pieces_base//100, load_supplier)
+    tables["region"] = (
+        "r",
+        1,
+        load_region,
+    )  # * dbgen only produces one file for region with SF1000 *
+    tables["supplier"] = ("s", num_pieces_base // 100, load_supplier)
```
Comment (alifarhid) [inline examples/tpch/generate_data_pq.py:50]: same as above, tuple entries in newlines.

Comment (alifarhid) [reply examples/tpch/generate_data_pq.py:50]: perhaps you ran black on these files and due to the comma at the end of load_region, it broke the entries into newlines.

File: examples/tpch/generate_data_pq.py [hunk 3]
```diff
@@ -52,7 +59,15 @@ def remove_file_if_exists(path):
...
-    SCALE_FACTOR, table_name, table_short, load_func, piece, num_pieces, output_prefix = args
+    (
+        SCALE_FACTOR,
+        table_name,
+        table_short,
+        load_func,
+        piece,
+        num_pieces,
+        output_prefix,
+    ) = args
```
Comment (alifarhid) [inline examples/tpch/generate_data_pq.py:70]: same here

File: examples/tpch/generate_data_pq.py [hunk 4]
```diff
@@ -67,7 +82,9 @@ def to_parquet(args):
     df.to_parquet(f"{output_prefix}/part-{zeros}{piece}.pq")
 
 
-def generate(tables, SCALE_FACTOR, folder, upload_to_s3, validate_dataset, num_processes):
+def generate(
+    tables, SCALE_FACTOR, folder, upload_to_s3, validate_dataset, num_processes
+):
```
Comment (alifarhid) [inline examples/tpch/generate_data_pq.py:88]: same

File: examples/tpch/generate_data_pq.py [hunk 5]
```diff
@@ -160,4 +179,6 @@ def generate(tables, SCALE_FACTOR, folder, upload_to_s3, validate_dataset, num_p
         num_pieces_base = 100
     tables = get_tables_info(num_pieces_base)
     set_start_method("spawn")
-    generate(tables, SCALE_FACTOR, folder, upload_to_s3, validate_dataset, num_processes)
+    generate(
+        tables, SCALE_FACTOR, folder, upload_to_s3, validate_dataset, num_processes
+    )
```
Comment (alifarhid) [inline examples/tpch/generate_data_pq.py:184]: same

File: examples/tpch/generate_data_pq.py [hunk 6]
```diff
@@ -128,7 +145,9 @@ def generate(tables, SCALE_FACTOR, folder, upload_to_s3, validate_dataset, num_p
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description="TPC-H Data Generation in Parquet Format")
+    parser = argparse.ArgumentParser(
+        description="TPC-H Data Generation in Parquet Format"
+    )
     parser.add_argument(
         "--SF",
         type=float,
```
Comment (alifarhid) [inline examples/tpch/generate_data_pq.py:153]: same

File: ml/README.md [hunk 7]
```diff
@@ -21,7 +21,7 @@ and paste the json text below and populate with your credentials.
 
 Open any of the notebooks and run it.
 
-watch your [htop](https://formulae.brew.sh/formula/htop) in terminal and enjoy your CPUs running at maximum efficiency.
+watch your [htop](https://formulae.brew.sh/formula/htop) in terminal and enjoy your CPUs running at full capacity.
```
Comment (alifarhid) [inline ml/README.md:24]: sometimes CPUs won't go beyond 70-80%, that's why we tried to avoid saying full capacity (100%).

**Review by alifarhid — CHANGES_REQUESTED** [review]
@ehsantn just some minor stylish comments. I guess black sometime broke into newlines which is not helping with readability.

File: ml/README.md (hunk 7 above)
Comment (ehsantn) [reply ml/README.md:24]: CPUs working is not the same as "maximum efficiency" so I'd be careful. I'll find a better word.

File: examples/tpch/generate_data_pq.py (hunk 1 above)
Comment (ehsantn) [reply examples/tpch/generate_data_pq.py]: `black` does this when the line is too long. I moved the comment above which fixed the issue.

File: examples/tpch/generate_data_pq.py (hunk 2 above)
Comment (ehsantn) [reply examples/tpch/generate_data_pq.py:44]: comma doesn't matter for `black`. The line was too long so I move the comment.

File: examples/tpch/generate_data_pq.py (hunk 3 above)
Comment (ehsantn) [reply examples/tpch/generate_data_pq.py:64]: `black` is trying to avoid long lines. This is worse than long lines?

File: ml/README.md [hunk 8]
```diff
@@ -21,7 +21,8 @@ and paste the json text below and populate with your credentials.
 
 Open any of the notebooks and run it.
 
-watch your [htop](https://formulae.brew.sh/formula/htop) in terminal and enjoy your CPUs running at maximum efficiency.
+Watch your [htop](https://formulae.brew.sh/formula/htop) in terminal and enjoy
+all your CPU cores working hard on ML problems!
```
Comment (ehsantn) [inline ml/README.md:25]: @alifarhid  Is this better?

Comment (alifarhid) [reply ml/README.md:25]: It's Ok @ehsantn . Just not sure why you added ML problems? On any data processing CPU cores are working hard.

**Review by alifarhid — APPROVED** [review]

**Pull request merged**